sycl/test-e2e/BFloat16/bfloat16_example_cpu.cpp

```diff
@@ -0,0 +1,18 @@
+///
+/// Check if bfloat16 example works using fallback libraries AOT compiled for
+/// CPU.
+///
+
+// REQUIRES: opencl-aot, ocloc, gpu-intel-gen12, cpu
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64,spir64_gen -Xsycl-target-backend=spir64_gen "-device pvc" %s -o %t.out
+// RUN: %{run} %t.out
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64_x86_64,spir64_gen -Xsycl-target-backend=spir64_gen "-device pvc" %s -o %t.out
+// RUN: %{run} %t.out
```

Comment:
`RUN` line is not guarded by `if`, so we will attempt to launch AOT-compiled code targeting PVC on a gen12 GPU. That will work thanks to AOT -> JIT fallback, but I don't think that we should do that